<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pablo Paiva - Sistemas Operacionais</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="trem_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/T1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trem_5">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>200</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/T2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trem_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/T4.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trem_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/T3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trem_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/T5.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1141</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/background.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="slider_trem1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #0000;
    height: 8px;
    background: red;
    margin: 2px 0;
}
QSlider::handle:horizontal {
    background:  rgb(240, 0, 0);
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0;
    border-radius: 3px;
}</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>40</number>
    </property>
   </widget>
   <widget class="QSlider" name="slider_trem2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #0000;
    height: 8px;
    background: rgb(220, 3, 205);
    margin: 2px 0;
}
QSlider::handle:horizontal {
    background:  rgb(220, 3, 205);
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0;
    border-radius: 3px;
}</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>40</number>
    </property>
   </widget>
   <widget class="QSlider" name="slider_trem3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #0000;
    height: 8px;
    background: green;
    margin: 2px 0;
}
QSlider::handle:horizontal {
    background:  green;
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0;
    border-radius: 3px;
}</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>40</number>
    </property>
   </widget>
   <widget class="QSlider" name="slider_trem5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #0000;
    height: 8px;
    background: blue;
    margin: 2px 0;
}
QSlider::handle:horizontal {
    background: blue;
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0;
    border-radius: 3px;
}</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>40</number>
    </property>
   </widget>
   <widget class="QSlider" name="slider_trem4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #0000;
    height: 8px;
    background: yellow;
    margin: 2px 0;
}
QSlider::handle:horizontal {
    background:  yellow;
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0;
    border-radius: 3px;
}</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>40</number>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>trem_1</zorder>
   <zorder>trem_5</zorder>
   <zorder>trem_3</zorder>
   <zorder>trem_4</zorder>
   <zorder>trem_2</zorder>
   <zorder>slider_trem1</zorder>
   <zorder>slider_trem2</zorder>
   <zorder>slider_trem3</zorder>
   <zorder>slider_trem5</zorder>
   <zorder>slider_trem4</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1140</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
